src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs

```diff
@@ -102,14 +102,11 @@ public static MeterProviderBuilder AddOtlpExporter(this MeterProviderBuilder bui
 
             var metricExporter = new OtlpMetricExporter(exporterOptions);
 
-            if (metricReaderOptions.MetricReaderType == (MetricReaderType)(-1))
-            {
-                metricReaderOptions.MetricReaderType = MetricReaderType.Periodic;
-            }
-
             var metricReader = metricReaderOptions.MetricReaderType == MetricReaderType.Manual
                 ? new BaseExportingMetricReader(metricExporter)
-                : new PeriodicExportingMetricReader(metricExporter, metricReaderOptions.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds);
+                : new PeriodicExportingMetricReader(
+                    metricExporter,
+                    metricReaderOptions.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds ?? 60000);
```

Comment:
Idea - Might be good to have the default outside of this extension in a constant somewhere; but I believe it is already defined somewhere else so using that is OK as well

Reply:
Moved to a constant. The constant already defined is part of the SDK project and is internal so not accessible here.

Reply:
Actually come to think of it the right thing to do here is to just do `new PeriodicExportingMetricReader(metricExporter);`. I've got a follow up PR to remove `MetricReaderType`. I'll clean that up in that PR.